.github/workflows/python.yml

```
@@ -204,9 +202,9 @@ jobs:
           args: |
             --manifest-path rerun_py/Cargo.toml
             --release
+            --target {{ matrix.target }}
```

```suggestion
            --target ${{ matrix.target }}
```
I think this needs $?